keps/sig-node/5573-cgroup-v1-unsupported/kep.yaml

```
@@ -0,0 +1,37 @@
+title: Move cgroup v1 to unsupported
+kep-number: 5573
+authors:
+  - "@kannon92"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: implementable
+creation-date: 2025-09-26
+reviewers:
+  - "@mrunalp"
+  - "@SergeyKanzhelev"
+  - "@yujuhong"
+approvers:
+  - "@derekwaynecarr"
```

```suggestion
  - "@mrunalp"
```

@mrunalp do you want to approve here instead of @derekwaynecarr ?

---

Sure

---

updated.